src/components/Datatable/Pagination/NavButton.tsx

```diff
@@ -0,0 +1,39 @@
+import styled from 'styled-components';
+
+import { getColor, pxToRem } from '../../../utils/helpers';
+import { IconButton } from '../../IconButton';
+import { IconButtonProps } from '../../IconButton/IconButton.types';
+
+const NavButton = styled(IconButton)
+  .attrs((props) => ({
+    ...props,
+  }))
+  .withConfig<IconButtonProps>({
+    shouldForwardProp: (prop) => prop !== 'disabled',
+  })`
+  display: inline-block;
+
+  width: ${pxToRem(48)};
+  height: ${pxToRem(32)};
+  padding: ${pxToRem(8, 16)};
+  margin: ${pxToRem(0, 4)};
+  border-radius: ${pxToRem(3)};
```

Comment:
we don't have 3px `border-radius`. We only use 4px (via `getBorderRadius` helper) or use `50em` (for pills like elements) or 100% (for rounded elements). If the 3px is used in Figma you should write to Mate, as it should be considered design error.